bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/TypeChecker.java

```diff
@@ -841,7 +841,8 @@ private static boolean checkObjectEquivalency(BType sourceType, BObjectType targ
         }
 
         for (AttachedFunction lhsFunc : targetFuncs) {
-            if (lhsFunc == targetType.initializer || lhsFunc == targetType.defaultsValuesInitFunc) {
+            if (lhsFunc == targetType.initializer || lhsFunc == targetType.defaultsValuesInitFunc
```

Comment:
do we need this anymore?

Reply:
We don't need that. so removed in 3875e4b